crates/bevy_core/Cargo.toml

```diff
@@ -19,4 +19,5 @@ bevy_tasks = { path = "../bevy_tasks", version = "0.8.0-dev" }
 bevy_utils = { path = "../bevy_utils", version = "0.8.0-dev" }
 
 # other
+async-channel = "1.4"
```

Comment:
Just curious, why have you chosen this particular channels crate as opposed to any of the other ones?

Reply:
It's already in our dependency tree. There wasn't much thought past that.

Reply:
as far as I can tell crossbeam_channel is not a dependency of async-channel https://github.com/smol-rs/async-channel/blob/master/Cargo.toml.

Reply:
looking into it some more we do have crossbeam channel as a dependency of bevy assets and are potentially picking it up in bevy winit https://github.com/bevyengine/bevy/pull/4726/files#diff-430ff15c09fecc41f366f88d7aac221cf60b43f87b36ed396673213aefa0ef16. I'm not sure what's preferred here. `bevy_ecs` is deeper in the dependency tree and depends on `async_channel`.  `bevy_winit` and `bevy_assets` are higher up in the dependency tree than `bevy_core`. But it probably makes more sense to use crossbeam since we don't use the async-ness of `async-channel`.

Reply:
switched to cross beam since we don't need the async-ness of async-channel